src/js/apps/patients/worklist/worklist_state.js

```diff
@@ -107,35 +110,66 @@ export default Backbone.Model.extend({
       [dateType]: this.formatDateRange(relativeRange, unit, dateFormat),
     };
   },
+  getDefaultStatesFilter() {
+    const isDoneOnly = this.id === 'done-last-thirty-days';
+    const { done, notDone } = this.states.groupByDone();
+
+    return isDoneOnly ? done : notDone;
+  },
+  setDefaultFilterStates() {
+    const defaultStatesFilter = this.getDefaultStatesFilter();
+    this.set({ filters: { states: defaultStatesFilter.map('id') } });
+  },
+  getSelectedStates() {
+    const defaultStatesFilterIds = this.getDefaultStatesFilter().map('id');
+    const selectedStates = this.getFilters().states;
+
+    // remove any invalid state ids (i.e. ids that don't belong to any default states)
+    return filter(selectedStates, id => contains(defaultStatesFilterIds, id)).join();
+  },
+  getFiltersCount() {
+    const filtersState = this.get('filters');
+    const defaultStatesFilter = this.getDefaultStatesFilter();
+
+    const customFiltersState = omit(filtersState, 'states');
```

Comment:
I think we should break this function up such that this ends up being
`return this.getCustomFiltersCount() + this.getStatesFilterCount();`

Reply:
Ah actually I realize now this method isn't being used.. yeah it should be on a different state model than this one who's class can be shared by both filter apps.